libs/http-parser/Makefile

```diff
@@ -1,36 +1,36 @@
 #
-# Copyright (C) 2013 OpenWrt.org
+# Copyright (C) 2013 - 2017 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=libhttp-parser
-PKG_VERSION:=2.3.0
+PKG_NAME:=http-parser
+PKG_VERSION:=2.7.1
 PKG_RELEASE=1
 PKG_MAINTAINER:=Ramanathan Sivagurunathan <ramzthecoder@gmail.com>
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE-MIT
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
```

Comment:
Ahh sorry, I read that you were still pulling it as git. Please disregard that comment, :-)